examples/ecs/query_bundle.rs

```diff
@@ -0,0 +1,62 @@
+use bevy::{
+    ecs::schedule::RunOnce,
+    log::{LogPlugin, LogSettings},
```

Comment:
```suggestion
    log::LogPlugin,
```
No longer necessary after refactor